README.md

```diff
@@ -151,6 +151,31 @@ for more info.
 To run rustfmt after this, use `cargo run --bin rustfmt -- filename`. See the
 notes above on running rustfmt.
 
+All command should be run in the project's root directory.
+
+### Update Toolchain
+
+Make sure to have installed the correct toolchain and all components.
+The needed parts can be found in the file `rust-toolchain`. For example:
+
+```
+[toolchain]
+channel = "nightly-2024-08-17"
+components = ["llvm-tools", "rustc-dev"]
+```
```

Comment:
I'd prefer not to list the content of the `rust-toolchain` file inline. Instead, I'd rather linked out to the file ``[`rust-toolchain`](https://github.com/rust-lang/rustfmt/blob/master/rust-toolchain)``, and let users know that they need to install the exact nightly version of the compiler that rustfmt has pinned in its `rust-toolchain` file.